jupyter-js-widgets/src/widget_box.js

```diff
@@ -229,34 +237,36 @@ var FlexBoxView = BoxView.extend({ // TODO: Deprecated in 5.0 (entire view)
     },
 
     update_orientation: function() {
-        var orientation = this.model.get("orientation");
-        if (orientation == "vertical") {
-            this.$box.removeClass("hbox").addClass("vbox");
+        var orientation = this.model.get('orientation');
+        if (orientation == 'vertical') {
+            this.box.classList.remove('hbox');
+            this.box.classList.add(vbox);
         } else {
-            this.$box.removeClass("vbox").addClass("hbox");
+            this.box.classList.remove('vbox');
+            this.box.classList.add('hbox');
```

Comment:
All the browsers that support WebSockets (i.e. our minimum browser) support `classList` *as long as* we add only one class at a time. Firefox supporse `element.classList.add('foo', 'bar', 'baz')` but IE does not.